File: knowledge/src/main/java/com/ayvytr/knowledge/adapter/GankHistoryAdapter.java
package com.ayvytr.knowledge.adapter;

import android.content.Context;

import com.ayvytr.baseadapter.ViewHolder;
import com.ayvytr.baseadapter.wrapper.EmptyWrapperAdapter;
import com.ayvytr.knowledge.R;

/**
 * @author admin
 */
public class GankHistoryAdapter extends EmptyWrapperAdapter<String> {
    public GankHistoryAdapter(Context context) {
        super(context, R.layout.layout_item_gank_history);
    }

    @Override
    public void convert(ViewHolder holder, String s, int position) {
        holder.setText(R.id.tv, s);
    }
}

File: knowledge/src/main/java/com/ayvytr/knowledge/adapter/GankHistoryAdapter.kt
package com.ayvytr.knowledge.adapter

import android.content.Context
import com.ayvytr.baseadapter.CommonAdapter
import com.ayvytr.baseadapter.ViewHolder
import com.ayvytr.knowledge.R

/**
 * @author admin
 */
class GankHistoryAdapter(context: Context) : CommonAdapter<String>(context, R.layout.layout_item_gank_history) {

    override fun convert(holder: ViewHolder, s: String, position: Int) {
        holder.setText(R.id.tv, s)
    }
}
